akka-cluster/src/main/resources/reference.conf

```diff
@@ -235,6 +235,8 @@ akka {
 
       # The n oldest nodes in a data center will choose to gossip to another data center with
       # this probability. Must be a value between 0.0 and 1.0 where 0.0 means never, 1.0 means always.
+      # When a data center is first started (nodes < 5) a higher probability is used so other data
+      # centers find out about the new nodes more quickly
```

Comment:
D'oh. I think Patrik said I should do this in the original PR but I guess it was never done. Good!